src/ui_empty.c

```diff
@@ -0,0 +1,54 @@
+#ifdef HAVE_BAGL
+#include "bolos_target.h"
+#include "globals.h"
+#include "ui.h"
+
+const bagl_element_t empty_screen_elements[] = {{{BAGL_RECTANGLE,
+                                                  BAGL_NONE,
+                                                  0,
+                                                  0,
+                                                  BAGL_WIDTH,
+                                                  BAGL_HEIGHT,
+                                                  0,
+                                                  0,
+                                                  BAGL_FILL,
+                                                  0x000000,
+                                                  0xFFFFFF,
+                                                  0,
+                                                  0},
+                                                 .text = NULL},
+                                                {}};
+
+typedef struct ux_layout_empty_params_s {
+    const char *line1;
```

Comment:
This `line1` is unused and can be removed. According to the `empty_screen_step` argument

Reply:
Removed.